### DISCUSSION TITLE
Trouble with snapshot music

### DISCUSSION THREAD

**Discussion by William4694** [exult/exult#327] [General]
I've been playing the official release on Mac OS for a while with no issues. When I install the snapshot, however, it doesn't seem to find the audio pack or ogg files. If I reinstall the official release, it finds the packs once again. Everything else remains the same between 1.8 and 1.9...savegames, mods, etc. Am I missing something on my end? Thanks!

**Comment by DominusExult** [comment]
I can confirm, thanks for reporting that!
I'll investigate later today.

**Comment by DominusExult** [comment]
Marzo pushed a fix and the snapshot is building right now and should be up in a few. Thanks for pointing this out!

**Comment by William4694** [comment]
Running brilliantly now...thank you! :D
